examples/features/nse-composition/README.md

```diff
@@ -0,0 +1,313 @@
+# Test NSE composition
```

Comment:
Please add the example into the features suite.

Reply:
ok, done